### ISSUE TITLE
【EESS】New submissions for Tue, 30 May 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#244]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### FoPro-KD: Fourier Prompted Effective Knowledge Distillation for  Long-Tailed Medical Image Recognition
 - **Authors:** Marawan Elbatel, Robert Martí, Xiaomeng Li
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2305.17421
 - **Pdf link:** https://arxiv.org/pdf/2305.17421
 - **Abstract**
 Transfer learning is a promising technique for medical image classification, particularly for long-tailed datasets. However, the scarcity of data in medical imaging domains often leads to overparameterization when fine-tuning large publicly available pre-trained models. Moreover, these large models are ineffective in deployment in clinical settings due to their computational expenses. To address these challenges, we propose FoPro-KD, a novel approach that unleashes the power of frequency patterns learned from frozen publicly available pre-trained models to enhance their transferability and compression. FoPro-KD comprises three modules: Fourier prompt generator (FPG), effective knowledge distillation (EKD), and adversarial knowledge distillation (AKD). The FPG module learns to generate targeted perturbations conditional on a target dataset, exploring the representations of a frozen pre-trained model, trained on natural images. The EKD module exploits these generalizable representations through distillation to a smaller target model, while the AKD module further enhances the distillation process. Through these modules, FoPro-KD achieves significant improvements in performance on long-tailed medical image classification benchmarks, demonstrating the potential of leveraging the learned frequency patterns from pre-trained models to enhance transfer learning and compression of large pre-trained models for feasible deployment.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Mobile Safety Application for Pedestrians
 - **Authors:** Sukru Yaren Gelbal, Mustafa Ridvan Cantas, Bilin Aksun Guvenc, Levent Guvenc, Gopichandra Surnilla, Hao Zhang
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2305.17575
 - **Pdf link:** https://arxiv.org/pdf/2305.17575
 - **Abstract**
 Vulnerable Road User (VRU) safety has been an important issue throughout the years as corresponding fatality numbers in traffic have been increasing each year. With the developments in connected vehicle technology, there are new and easier ways of implementing Vehicle to Everything (V2X) communication which can be utilized to provide safety and early warning benefits for VRUs. Mobile phones are one important point of interest with their sensors being increased in quantity and quality and improved in terms of accuracy. Bluetooth and extended Bluetooth technology in mobile phones has enhanced support to carry larger chunks of information to longer distances. The work we discuss in this paper is related to a mobile application that utilizes the mobile phone sensors and Bluetooth communication to implement Personal Safety Message (PSM) broadcast using the SAE J2735 standard to create a Pedestrian to Vehicle (P2V) based safety warning structure. This implementation allows the drivers to receive a warning on their mobile phones and be more careful about the pedestrian intending to cross the street. As a result, the driver has much more time to safely slow down and stop at the intersection. Most importantly, thanks to the wireless nature of Bluetooth connection and long-range mode in Bluetooth 5.0, most dangerous cases such as reduced visibility or No-Line-of-Sight (NLOS) conditions can be remedied.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Rate-Distortion Theory in Coding for Machines and its Application
 - **Authors:** Alon Harell, Yalda Foroutan, Nilesh Ahuja, Parual Datta, Bhavya Kanzariya, V. Srinivasa Somayaulu, Omesh Tickoo, Anderson de Andrade, Ivan V. Bajic
 - **Subjects:** Image and Video Processing (eess.IV); Information Theory (cs.IT)
 - **Arxiv link:** https://arxiv.org/abs/2305.17295
 - **Pdf link:** https://arxiv.org/pdf/2305.17295
 - **Abstract**
 Recent years have seen a tremendous growth in both the capability and popularity of automatic machine analysis of images and video. As a result, a growing need for efficient compression methods optimized for machine vision, rather than human vision, has emerged. To meet this growing demand, several methods have been developed for image and video coding for machines. Unfortunately, while there is a substantial body of knowledge regarding rate-distortion theory for human vision, the same cannot be said of machine analysis. In this paper, we extend the current rate-distortion theory for machines, providing insight into important design considerations of machine-vision codecs. We then utilize this newfound understanding to improve several methods for learnable image coding for machines. Our proposed methods achieve state-of-the-art rate-distortion performance on several computer vision tasks such as classification, instance segmentation, and object detection.
### Trustworthy Deep Learning for Medical Image Segmentation
 - **Authors:** Lucas Fidon
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2305.17456
 - **Pdf link:** https://arxiv.org/pdf/2305.17456
 - **Abstract**
 Despite the recent success of deep learning methods at achieving new state-of-the-art accuracy for medical image segmentation, some major limitations are still restricting their deployment into clinics. One major limitation of deep learning-based segmentation methods is their lack of robustness to variability in the image acquisition protocol and in the imaged anatomy that were not represented or were underrepresented in the training dataset. This suggests adding new manually segmented images to the training dataset to better cover the image variability. However, in most cases, the manual segmentation of medical images requires highly skilled raters and is time-consuming, making this solution prohibitively expensive. Even when manually segmented images from different sources are available, they are rarely annotated for exactly the same regions of interest. This poses an additional challenge for current state-of-the-art deep learning segmentation methods that rely on supervised learning and therefore require all the regions of interest to be segmented for all the images to be used for training. This thesis introduces new mathematical and optimization methods to mitigate those limitations.
### Stochastic Pitch Prediction Improves the Diversity and Naturalness of  Speech in Glow-TTS
 - **Authors:** Sewade Ogun, Vincent Colotte, Emmanuel Vincent
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2305.17724
 - **Pdf link:** https://arxiv.org/pdf/2305.17724
 - **Abstract**
 Flow-based generative models are widely used in text-to-speech (TTS) systems to learn the distribution of audio features (e.g., Mel-spectrograms) given the input tokens and to sample from this distribution to generate diverse utterances. However, in the zero-shot multi-speaker TTS scenario, the generated utterances lack diversity and naturalness. In this paper, we propose to improve the diversity of utterances by explicitly learning the distribution of fundamental frequency sequences (pitch contours) of each speaker during training using a stochastic flow-based pitch predictor, then conditioning the model on generated pitch contours during inference. The experimental results demonstrate that the proposed method yields a significant improvement in the naturalness and diversity of speech generated by a Glow-TTS model that uses explicit stochastic pitch prediction, over a Glow-TTS baseline and an improved Glow-TTS model that uses a stochastic duration predictor.
### A Semi-Supervised Learning Approach for Ranging Error Mitigation Based  on UWB Waveform
 - **Authors:** Yuxiao Li, Santiago Mazuelas, Yuan Shen
 - **Subjects:** Signal Processing (eess.SP); Artificial Intelligence (cs.AI); Machine Learning (cs.LG); Applications (stat.AP)
 - **Arxiv link:** https://arxiv.org/abs/2305.18208
 - **Pdf link:** https://arxiv.org/pdf/2305.18208
 - **Abstract**
 Localization systems based on ultra-wide band (UWB) measurements can have unsatisfactory performance in harsh environments due to the presence of non-line-of-sight (NLOS) errors. Learning-based methods for error mitigation have shown great performance improvement via directly exploiting the wideband waveform instead of handcrafted features. However, these methods require data samples fully labeled with actual measurement errors for training, which leads to time-consuming data collection. In this paper, we propose a semi-supervised learning method based on variational Bayes for UWB ranging error mitigation. Combining deep learning techniques and statistic tools, our method can efficiently accumulate knowledge from both labeled and unlabeled data samples. Extensive experiments illustrate the effectiveness of the proposed method under different supervision rates, and the superiority compared to other fully supervised methods even at a low supervision rate.
### Experimental Demonstration of Stokes Space Equalization for Space  Division Multiplexed Signals
 - **Authors:** F.J. Vaquero-Caballero, Gernot Goeger, Fabio Pittala, Yabin Ye, Idelfonso Tafur Monroy
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2305.18220
 - **Pdf link:** https://arxiv.org/pdf/2305.18220
 - **Abstract**
 In this letter we experimentally validate, for the first time, the Stokes space algorithm (SSA) equalizer for space division multiplexing (SDM) transmission systems. We introduce the frequency domain (FD)-SSA and FD least-mean square (FD-LMS) algorithms, and evaluate their performance for different frequency offsets by computer simulations. Our simulations show that FD-SSA is insensitive to the frequency offsets, not requiring carrier frequency estimation (CFE) before equalization (pre-CFE) or carrier phase estimation (CPE) inside the loop (L-CPE). Our experimental results confirm that FD-SSA presents the same performance as FD-LMS, where the required digital signal processing (DSP) stack for FD-SSA is simpler compared to the FDLMS.
### Temporal Aware Mixed Attention-based Convolution and Transformer Network  (MACTN) for EEG Emotion Recognition
 - **Authors:** Xiaopeng Si, Dong Huang, Yulin Sun, Dong Ming
 - **Subjects:** Signal Processing (eess.SP); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2305.18234
 - **Pdf link:** https://arxiv.org/pdf/2305.18234
 - **Abstract**
 Emotion recognition plays a crucial role in human-computer interaction, and electroencephalography (EEG) is advantageous for reflecting human emotional states. In this study, we propose MACTN, a hierarchical hybrid model for jointly modeling local and global temporal information. The model is inspired by neuroscience research on the temporal dynamics of emotions. MACTN extracts local emotional features through a convolutional neural network (CNN) and integrates sparse global emotional features through a transformer. Moreover, we employ channel attention mechanisms to identify the most task-relevant channels. Through extensive experimentation on two publicly available datasets, namely THU-EP and DEAP, our proposed method, MACTN, consistently achieves superior classification accuracy and F1 scores compared to other existing methods in most experimental settings. Furthermore, ablation studies have shown that the integration of both self-attention mechanisms and channel attention mechanisms leads to improved classification performance. Finally, an earlier version of this method, which shares the same ideas, won the Emotional BCI Competition's final championship in the 2022 World Robot Contest.
